src/main/webapp/i18n/de/course.json

```
@@ -90,6 +90,27 @@
             "upload": "Hochladen",
             "uploading": "Lade hoch...",
             "fileUploadExercises": "Dateiupload-Aufgabe",
+            "search": {
+                "searchButton": "Suchen",
+                "toggleButton": {
+                    "true": "Zeige erweiterte Suche",
+                    "false": "Verstecke erweiterte Suche"
+                },
+                "cardTitle": "Aufgaben durchsuchen",
+                "fieldExerciseName": "Aufgaben nach Namen durchsuchen",
+                "labelExerciseCategory": "Kategorie",
+                "fieldExerciseCategory": "Kategorie",
+                "labelExerciseType": "Aufgabentyp",
+                "noResults": "Keine Aufgaben wurden für die Suchparameter gefunden",
+                "type": {
+                    "all": "Alle Aufgaben",
+                    "programming": "Programmieraufgaben",
+                    "quiz": "Quizaufgaben",
+                    "modeling": "Modellierungsaufgaben",
+                    "text": "Textaufgabe",
+                    "file-upload": "Dateiupload-Aufgabe"
```

```suggestion
                    "text": "Textaufgaben",
                    "file-upload": "Dateiupload-Aufgaben"
```
Other translations and English version are plural.